P5/Source/Specs/post.xml

```diff
@@ -0,0 +1,252 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium.
+Dual-licensed under CC-by and BSD2 licences
+See the file COPYING.txt for details.
+-->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec xmlns="http://www.tei-c.org/ns/1.0" ident="post" mode="add" module="cmc">
+    <desc xml:lang="en" versionDate="2022-08-09"> a written (or spoken) contribution to an ongoing
```

Comment:
Why is a post always a contribution to an "_ongoing_ CMC interaction"? It also can _start_ one?!

Reply:
Fixed in 759c2adba.